src/include/cpp-btree/btree.h

```diff
@@ -863,6 +863,30 @@ struct btree_iterator {
   int position = -1;
 };
 
+template <size_t Alignment, class Alloc>
+class AlignedAlloc {
+  struct alignas(Alignment) M {};
+  using alloc_t =
+    typename std::allocator_traits<Alloc>::template rebind_alloc<M>;
+  using traits_t =
+    typename std::allocator_traits<Alloc>::template rebind_traits<M>;
+  static constexpr size_t round_up(size_t n) {
```

Comment:
```suggestion
  static constexpr size_t objects_needed(size_t n) {
```
I was just confused when "round_up" changed domain from bytes to object count.

Reply:
@aclamk i 

- s/n/size/
- s/round_up/num_aligned_objects/ instead of using "objects_needed". because i think it is more consistent with the naming of "aligned_alloc". and help posterities to understand this context: we are trying to enforce the alignment here.